bitnami/fluentd/templates/serviceaccount.yaml

```diff
@@ -4,4 +4,7 @@ kind: ServiceAccount
 metadata:
   name: {{ include "fluentd.serviceAccountName" . }}
   labels: {{- include "fluentd.labels" . | nindent 4 }}
-{{- end -}}
\ No newline at end of file
+  {{- if .Values.serviceAccount.annotations }}
+  annotations: {{ toYaml .Values.serviceAccount.annotations | nindent 4 }}
```

Comment:
Could we follow this approach using `fluentd.tplValue`?

https://github.com/bitnami/charts/blob/master/bitnami/fluentd/templates/aggregator-statefulset.yaml#L23